tendermint/src/lite.rs

```diff
@@ -4,4 +4,9 @@ pub mod types;
 pub mod verifier;
 
 pub use self::types::*;
-pub use self::verifier::*;
+
+// TODO: don't expose this once the json tests
+// switch to using one of the other functions
```

Comment:
Do we have a follow-up for this in our meta issues?

Reply:
Yes its in #110